pack/Makefile

```diff
@@ -14,7 +14,7 @@ else ifneq (,$(wildcard /etc/debian_version))
 include $(PACKDIR)/deb.mk
 else ifneq (,$(wildcard /etc/alpine-release))
 include $(PACKDIR)/apk.mk
-else ifneq (,$(shell grep "^ID=\"opensuse-leap\"" /etc/os-release))
+else ifneq (,$(shell grep "opensuse" /etc/os-release))
```

Comment:
It would be nice if the commit (and the pull request description) reveals details about the target OS and the needed changes. At least I would like to know a content of the `/etc/os-release` file on this OS.

Reply:
Fixed (added separate condition to check against MosOS-Arbat distribution).

Reply:
Now it looks quite straightforward, I'm OK. Thanks!